cmd/kola/options.go

```diff
@@ -51,6 +53,7 @@ func init() {
 	root.PersistentFlags().IntVarP(&kola.TestParallelism, "parallel", "j", 1, "number of tests to run in parallel")
 	sv(&kola.TAPFile, "tapfile", "", "file to write TAP results to")
 	sv(&kola.Options.BaseName, "basename", "kola", "Cluster name prefix")
+	sv(&debugSystemdUnits, "debug-systemd-units", "", "space delimited list of full-unit-names.service to enabled SYSTEMD_LOG_LEVEL=debug on")
```

Comment:
`to enable`

This should probably also use a string slice variable (e.g. `--debug-systemd-unit systemd-networkd.service --debug-systemd-unit systemd-resolved.service`) rather than using internal delimiters in the argument.